tests/output/LoggingTest.py

```diff
@@ -3,6 +3,17 @@
 import unittest
 
 from coalib.output.Logging import configure_json_logging
+import coalib.output.printers.LogHelper as LogHelper
+from coalib.output.printers.LogPrinterMixin import LogPrinterMixin
+
+
+class TestLogger(LogPrinterMixin):
```

Comment:
Yeah, we do.
I've added it to cover the line https://github.com/coala/coala/pull/4320/files/c367b4ca2e3157746a51ed3ad13abdbfeabb21ec#diff-16aa1ce38a1d20ae5cc6c79934589eb2R45. (`LogPrinterMixin.log_message()` definition is not called due to inheritance, and every time base class' `log_message` function is executed)
May be I could have done it by calling `LogPrinterMixin().log_message(...)`, but this seemed a bit cleaner way.